release.config.cjs

```diff
@@ -0,0 +1,44 @@
+const transform = require('./release.utils.cjs');
+/**
+ * @type {import('semantic-release').GlobalConfig}
+ */
+module.exports = {
+    branches: ['master'],
```

Comment:
Not sure if we'll need `pre-release` branches. I'll add them incase we need them.
```suggestion
    branches: ['master'],
```